### PR TITLE
Webhooks module

### PR DESCRIPTION
**Pull request by Skrypt**
## TODO

- [ ] Make a list of things to add based on comments made on the original PR from @jrestall 
- [ ] Fix LiquidWebHookExpressionEvaluator Liquid template rendering.

### CODE REVIEW

**Comment by Skrypt** [comment]
@jrestall Feel free to contribute on this one.

**Comment by jrestall** [comment]
@Skrypt I most likely won't have time, but it's good to see the WebHooks module potentially being used. The initial feedback was it duplicated too much of what the Workflows module could do, but I still think it's nice to have straightforward WebHooks support in OC.

**Comment by Skrypt** [comment]
I like the fact that it's simpler than creating a Workflow. Sometimes all you need is a Webhook. I will try to analyze what you did here and come up with ideas.

**Comment by jrestall** [comment]
I think I based it on Contentful's Webhook support.

https://www.contentful.com/developers/docs/concepts/webhooks/

File: src/OrchardCore.Modules/OrchardCore.WebHooks/Models/ContentEvents.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+        public static string Updated = "updated";
+
+        public static string Removed = "removed";
+
+        public static string Published = "published";
+
+        public static string Unpublished = "unpublished";
+
+        public static IEnumerable<string> AllEvents => new[]
```
Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.WebHooks/Models/ContentEvents.cs:17]:
```suggestion
        public static string[] AllEvents => new[]
```

**Comment by Skrypt** [comment]
@piedone At some point I had the intent to rework on that one but never found time. Up to you to decide if we should close the PR or not. Normally, I would just keep it as a draft for future reference.

**Comment by Piedone** [comment]
I see, let's close this. If you still think there is value in having a separate webhook module in addition to Workflows, then please open an issue an link this PR.

**Pull request closed without merging**